wolfssl/openssl/aes.h

```diff
@@ -42,32 +42,7 @@
  * OpenSSL compatibility layer. This makes code working with an AES structure
  * to need the size of the structure. */
 typedef struct WOLFSSL_AES_KEY {
-    /* aligned and big enough for Aes from wolfssl/wolfcrypt/aes.h */
-    ALIGN16 void* holder[(376 + WC_ASYNC_DEV_SIZE)/ sizeof(void*)];
-    #ifdef GCM_TABLE
-    /* key-based fast multiplication table. */
-    ALIGN16 void* M0[4096 / sizeof(void*)];
-    #elif defined(GCM_TABLE_4BIT)
-    #if !defined(WORD64_AVAILABLE) || defined(BIG_ENDIAN_ORDER)
-    ALIGN16 byte M0[16][AES_BLOCK_SIZE];
-    #else
-    ALIGN16 byte M0[32][AES_BLOCK_SIZE];
-    #endif
-    #endif /* GCM_TABLE */
-    #if defined(WOLFSSL_DEVCRYPTO) && \
-        (defined(WOLFSSL_DEVCRYPTO_AES) || defined(WOLFSSL_DEVCRYPTO_CBC))
-    /* large enough for additional devcrypto information */
-    void* devKey[288 / sizeof(void*)];
-    #endif
-    #ifdef WOLFSSL_AFALG
-    void* afalg_holder[288 / sizeof(void*)];
-    #endif
-    #ifdef HAVE_PKCS11
-    void* pkcs11_holder[(AES_MAX_ID_LEN + sizeof(int)) / sizeof(void*)];
-    #endif
-    #if defined(WOLFSSL_ASYNC_CRYPT) || defined(WOLF_CRYPTO_CB)
-    void* async_holder[128 / sizeof(void*)];
-    #endif
+    void *buf[(sizeof(Aes) / sizeof(void *)) + 1];
```

Comment:
16byte alignment was required on some platforms with some builds.  "void*" is not guaranteed to have 16byte alignment.  Why was this removed?

Reply:
Refactored because `async_holder` was smaller than `Aes` with some configs, i.e. buffer overrun bug.

The new code has identical alignment dynamics to the old code as far as I can see.  Array of `(void *)` before and after.

Reply:
OIC, the github-generated excerpt above doesn't show the original code.  `ALIGN16` is indeed in the original and I'll restore it and get that pushed momentarily.

Thanks for catching this.